backend/src/Data/SeedData.cs

```diff
@@ -31,5 +31,43 @@ public static void Seed(ModelBuilder builder)
                 Organization = ""
             }
         });
+
+        var componentList = new List<Component>
+        {
+            new Component()
```

Comment:
I am not sure but I think it is enough here with `new()` and no need to repeat Component

Reply:
Fixed :blush: